mediapipe/calculators/util/multi_face_landmarks_smoothing_calculator.cc

```diff
@@ -0,0 +1,491 @@
+// Copyright 2020 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
What was changed in this file (is it a copy of some existing MediaPipe smoothing calculator)?

Reply:
yes, the smoothing by default is only for one face and don't work with attention model, so I copy that graph and change a little